app/scripts/modules/core/src/notification/types/githubstatus/githubstatus.notification.type.module.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.core.notification.types.githubstatus', [])
+  .config(function(notificationTypeConfigProvider) {
+    notificationTypeConfigProvider.registerNotificationType({
+      label: 'Github Status',
+      key: 'githubStatus',
```

Comment:
This is the only camel cased `githubStatus` - is this one wrong?

Reply:
No, it's the notification type in echo: https://github.com/spinnaker/echo/pull/344/files#diff-59ef91a5c716e1ef92630c9252135c89R131